Bowling/Bowling/Bowling.cs

```diff
@@ -0,0 +1,105 @@
+﻿using NUnit.Framework.Constraints;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace BowlingApp
+{
+    public class Bowling
+    {
+        public const int frameNumberTotal = 10;   // total number of frames in a bowling game
+
+        public void stepsForGameCalculation()
+        {
+            //create screen banner
+            createBanner();
+            
+            //get bowling marks from user
+            string bowlingMarks = getBowlingMarks();
+
+            // check to make sure we have values or end calculation
+            if(!bowlingMarks.Equals("0"))
+            {
+                Frames frames = new Frames();
+                
+                // use input string to create a list containing each bowling frames
+                List<Frames> allFrames = frames.CreateFrames(bowlingMarks);
+
+                // the scores for each frame now need to be updated to reflect any additional points for strikes and spares.
+                allFrames = frames.UpdateFrameScores(allFrames);
+
+                // calculate grand total
+                int grandTotal = calculateGrandTotal(allFrames);
+
+                // display grand total
+                Console.WriteLine(String.Concat("Final Bowling Score is : ", grandTotal));
```

Comment:
For bonus points, check out [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated)